include/gepetto/gui/config-dep.hh.in

```diff
@@ -2,5 +2,6 @@
 # define GEPETTO_GUI_CONFIG_DEP_HH
 
 #cmakedefine01 GEPETTO_GUI_HAS_PYTHONQT
+#cmakedefine AVCONV "@AVCONV@"
```

Comment:
I would prefer adding `TARGET_COMPILE_DEFINITIONS(${LIBRARY_NAME} PRIVATE -D'AVCONV="${AVCONV}"')` in src/CMakeLists.txt.